sdk/azidentity/azure_cli_credential.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azidentity
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+// Enables authentication to Azure Active Directory using Azure CLI to generated an access token.
+type AzureCLICredential struct {
+	Client      *azureCLICredentialClient
+	shellClient shellClient
+}
+
+// AzureCLICredentialOptions contains the parameters for the ShellClient to execute the Azure CLI command that will return an Access Token.
+type AzureCLICredentialOption struct {
```

Comment:
*Options (add 's' suffix)

Reply:
Thanks for your review, We are looking into those comments carefully . As we were notified that @schabbs is finalize the design. We will resolve all comments later.